### PR TITLE
LPS-181936

### PR DESCRIPTION
**Pull request by CaiocSiqueira**
https://issues.liferay.com/browse/LPS-181936

Original pull request: https://github.com/CaiocSiqueira/liferay-portal/pull/32

### CODE REVIEW

**Comment by CaiocSiqueira** [comment]
ci:test:sf

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase [hunk 1]
```diff
@@ -563,6 +563,91 @@ definition {
 		Alert.viewErrorMessage(errorMessage = "Your request failed to complete.");
 	}
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase:565]: @description = "This is the use case for LPS-181668."

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase [hunk 2]
```diff
@@ -563,6 +563,91 @@ definition {
...
 
+	@priority = 3
+	test CanPublishAPublicationAfterAPageHasBeenDeleted {
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase:567]: Name the test `CanPublishPublicationAfterPageHasBeenDeleted`. Usually we don't add articles (e.g. a, an, the) to  titles.

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase [hunk 3]
```diff
@@ -563,6 +563,91 @@ definition {
...
+				gotoPublicationsAdmin = "true",
+				publicationName = "Publication Name");
+
+			Button.clickPublish();
+
+			AssertTextEquals.assertPartialText(
+				locator1 = "Message#SUCCESS",
+				value1 = "No unresolved conflicts, ready to publish.");
+
+			Publications.publishPublication(publicationName = "Publication Name");
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase:647]: Add an assertion after every major action
```
Publications.viewPublicationPublished(publicationName = "Publication Name");
```

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase [hunk 4]
```diff
@@ -563,6 +563,91 @@ definition {
...
+		}
+
+		task ("Delete copied page and try to publish the publication") {
+			PagesAdmin.openPagesAdmin(siteURLKey = "test-site-name");
+
+			PagesAdmin.deletePage(pageName = "Home");
+
+			PublicationsNavigator.gotoReviewChanges(
+				gotoPublicationsAdmin = "true",
+				publicationName = "Publication Name");
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase:639]: Are these variables needed?

File: portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase [hunk 5]
```diff
@@ -563,6 +563,91 @@ definition {
...
+
+			JSONLayout.addPublicLayout(
+				groupName = "Test Site Name",
+				layoutName = "Home",
+				type = "content");
+
+			JSONLayout.publishLayout(
+				groupName = "Test Site Name",
+				layoutName = "Home");
+		}
```
Comment (xbrianlee) [inline portal-web/test/functional/com/liferay/portalweb/tests/enduser/publications/Publications.testcase:579]: Use the home page on the default site. No need to create a new site for this test.

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by CaiocSiqueira** [comment]
ci:test:publications:nocompile

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by CaiocSiqueira** [comment]
ci:test:publications:nocompile

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Comment by CaiocSiqueira** [comment]
ci:test:publications:nocompile

**Comment by CaiocSiqueira** [comment]
ci:test:sf

**Pull request closed without merging**